crates/types/src/config/common.rs

```diff
@@ -480,6 +480,33 @@ pub struct MetadataStoreClientOptions {
     pub metadata_store_client_backoff_policy: RetryPolicy,
 }
 
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+#[serde(
+    tag = "type",
+    rename_all = "kebab-case",
+    rename_all_fields = "kebab-case"
+)]
+#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
+#[cfg_attr(
+    feature = "schemars",
+    schemars(
+        title = "Metadata Store",
+        description = "Definition of a bootstrap metadata store"
+    )
+)]
+pub enum ObjectStoreCredentials {
+    /// # Use standard AWS environment variables
+    ///
+    /// configure the object store by setting the standard AWS env variables (prefixed with AWS_)
+    /// for example, to test with minio:
+    /// * AWS_ALLOW_HTTP="true"
+    /// * AWS_PROVIDER_NAME=Static
+    /// * AWS_ENDPOINT=http://127.0.0.1:9000
+    /// * AWS_ACCESS_KEY_ID="minioadmin"
+    /// * AWS_SECRET_ACCESS_KEY="minioadmin"
```

Comment:
This part will be added to our docs once we generate the config docs. Is this something we want to keep or is it for internal testing purposes?